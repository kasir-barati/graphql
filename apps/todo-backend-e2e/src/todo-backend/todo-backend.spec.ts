import axios from 'axios';

describe('POST /graphql', () => {
  it('should get a todo', async () => {
    const query = /* GraphQL */ `
      {
        getTodo(id: "eec5e3aa-7137-4c9c-a723-ec2f43d4daa4") {
          id
          content
        }
      }
    `;

    const res = await axios.post(`/graphql`, { query });

    console.log(res.data);

    expect(res.status).toBe(200);
    expect(res.data).toMatchSnapshot();
  });
});
