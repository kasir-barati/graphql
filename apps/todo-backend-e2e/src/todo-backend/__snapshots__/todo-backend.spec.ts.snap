// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`POST /graphql should get a todo 1`] = `
{
  "data": {
    "getTodo": {
      "content": "Some content",
      "id": "eec5e3aa-7137-4c9c-a723-ec2f43d4daa4",
    },
  },
}
`;
